create_funcs/create_player.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+from data_from_players.get_player_data import get_player_data
+
+from db.models import Player
+from players_json import players_data
+
+
+def create_player() -> Player:
+    player_lst = get_player_data(players_data)
+    for player in player_lst:
+        print(player)
+        Player.objects.get_or_create(
+            nickname=player[0],

```

Comment:
it is better to use key instead of index

Reply:
Used keys.